services/docker-compose.yml

```diff
@@ -113,6 +113,35 @@ services:
       - interactive_services_subnet
       - computational_services_subnet
 
+  static-webserver:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/static-webserver:${DOCKER_IMAGE_TAG:-latest}
+    environment:
+      - SERVER_HOST=0.0.0.0
+      - SERVER_PORT=80
+      - SERVER_LOG_LEVEL=error
+      - SERVER_ROOT=/static-content
+    env_file:
+      - ../.env
+    deploy:
+      placement:
+        constraints:
+          - node.platform.os == linux
+      labels:
+        - io.simcore.zone=${TRAEFIK_SIMCORE_ZONE}
+        - traefik.http.middlewares.${SWARM_STACK_NAME}_gzip.compress=true
+        - traefik.enable=true
+        - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.server.port=80
+        - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.healthcheck.path=/
+        - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.healthcheck.interval=2000ms
+        - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.healthcheck.timeout=1000ms
+        - traefik.http.middlewares.${SWARM_STACK_NAME}_static_webserver_retry.retry.attempts=2
+        - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.rule=hostregexp(`{host:.+}`) && (PathPrefix(`/osparc`) || PathPrefix(`/s4l`) || PathPrefix(`/tis`) || PathPrefix(`/transpiled`) || PathPrefix(`/resource`))
```

Comment:
Inserting these rows in the products table,  
```sql
INSERT INTO "products" ("name", "host_regex") VALUES
('s4l', '(^s4l[\.-])|(^sim4life\.)|(^api.s4l[\.-])|(^api.sim4life\.)'),
('tis', '(^tis[\.-])|(^ti-solutions\.)');
```
I still not get the corresponding front-end ... by http://s4l-127.0.0.1.nip.io:9081 (NOTE in your description there is a DOT missing). It returns always osparc front-end

Reply:
> I know you are already aware, but this is what you need in order to see the correct frontends.
> 
> ![Screenshot 2021-05-26 at 15 49 05](https://user-images.githubusercontent.com/5694077/119671242-e89dc880-be39-11eb-8296-22b65a3e942a.png)

And how different is from what I wrote above? Please read again the NOTE

Reply:
Ohh I know why!  You might have added the products to the database when the webserver was already running. 
You need to restart the webserver because the index.html pages are fetched on boot.